### DISCUSSION TITLE
Increase solution resolution

### DISCUSSION THREAD

**Discussion by offusername** [pybamm-team/PyBaMM#3460] [Q&A]
Hi together, 
how can i increase the sim.solution resolution? As you can see in the plot the graph has a quit high resolution. I tried to reduce the dt_max for the CasadiSolver. The time = sim.solution.t vector includes only 13 entrances.
May you can help me out how to refine the resulting vectors. 
BR Christopher
![DCH_5C](https://github.com/pybamm-team/PyBaMM/assets/141131323/4468249b-a32a-453d-b704-370991319461)

**Comment by valentinsulzer** [comment]
If you are using an `Experiment`, you can change the `period` argument

**Reply by offusername** [reply]
Many thanks @tinosulzer for guiding to the experiment - i was so focused on solver properties :)
